tests/__init__.py

```diff
@@ -22,15 +22,5 @@
 from __future__ import absolute_import
 
 from .api import *
-from .configuration import *
```

Comment:
any reason we remove these import lines?

Reply:
@Fokko  @XD-DENG

Reply:
Hi @feng-tao , I was travelling in the past day and didn't have network.

For more details, you can refer to JIRA ticket https://issues.apache.org/jira/browse/AIRFLOW-3239, and another two PRs (https://github.com/apache/incubator-airflow/pull/4049, https://github.com/apache/incubator-airflow/pull/4074)

Some history: 
- Firstly, I found some tests were never run since they were not named correctly. Test discovery process fails to find them (like tests for DockerOperator before https://github.com/apache/incubator-airflow/pull/4049).
- Some test files were not named correctly while they were still run, because of usage like `from .configuration import *` in `tests/__init__.py`, while this may not be the most proper method as @ashb shared above.

Any import lines removed in `tests/__init__.py` come together with a test file renaming like `tests/configuration.py → tests/test_configuration.py`.

Please let me know if you need any further clarification. Thanks.

Reply:
Thanks for the info.